javalib/src/main/scala/java/lang/Throwables.scala

```diff
@@ -11,6 +11,7 @@ import scala.scalanative.runtime.Backtrace
 import scala.util.Try
 import scala.util.Failure
 import scala.util.Success
+import scala.scalanative.runtime.Platform
```

Comment:
Instead of `Platform` can we use a linktime condition?
- https://github.com/scala-native/scala-native/pull/3335

Reply:
Thanks! Changed to use linktime condition https://github.com/scala-native/scala-native/pull/3343/commits/11d3e465cfc872244bbd5e49d22ad843d5982921 :+1: